test/Microsoft.AspNetCore.SignalR.Tests.Utils/ServerFixture.cs

```diff
@@ -223,9 +225,43 @@ public bool IsEnabled(LogLevel logLevel)
 
             public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
             {
-                _logSinkProvider.Log(_categoryName, logLevel, eventId, state, exception, formatter);
+                var connectionId = GetConnectionId();
+
+                Func<TState, Exception, string> newFormatter = (s, e) =>
+                {
+                    StringBuilder sb = new StringBuilder();
+                    if (connectionId != null)
```

Comment:
Good thing this is test code or @davidfowl would be exploding about the capture ;)